docs/contributors.yaml

```diff
@@ -402,13 +402,14 @@
   jira_user: yanyanhu
   reviewboard_user: yanyanhu
 
-- name: Yong Qiao Wang
```

Comment:
@vinodkone I also used `gradywang` as my reviewboard username  rather than `JamesYongQiaoWang` when I in IBM in main contributions. For my name, it does not changed, and just change to the recommended format. So can I only change the old entry?